### PR TITLE
add helm chart for signed url server

### PR DESCRIPTION
**Pull request by nikp1172**

### CODE REVIEW

File: charts/tfy-workflow-propeller/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  Create the env file
+  */}}
+{{- define "tfy-signed-url-server.env" }}
+{{- range $key, $val := (include "tfy-signed-url-server.parseEnv" .) | fromYaml }}
+{{- if and $val (contains "${k8s-secret" ($val | toString)) }}
+{{- if eq (regexSplit "/" $val -1 | len) 2 }}
+- name: {{ $key }}
+  valueFrom:
+    secretKeyRef:
+      name: {{ $.Values.tfySignedUrlServer.envSecretName }}
```
Comment (akashg3627) [inline charts/tfy-workflow-propeller/templates/_helpers.tpl:81]: define `envSecretName` in values

File: charts/tfy-workflow-propeller/values.yaml [hunk 2]
```diff
@@ -91,3 +91,77 @@ flyte-core:
...
   cluster_resource_manager:
     enabled: false
+
+tfySignedUrlServer:
+  ## @param tfySignedUrlServer.enabled to enable the tfySignedUrlServer
+  enabled: true
+  ## @param tfySignedUrlServer.serviceAccountName to set the service account name
+  serviceAccountName: flytepropeller
+  ## @param tfySignedUrlServer.tolerations to set the tolerations
+  tolerations: {}
```
Comment (akashg3627) [inline charts/tfy-workflow-propeller/values.yaml:101]: this is usually a array

File: charts/tfy-workflow-propeller/values.yaml [hunk 3]
```diff
@@ -91,3 +91,77 @@ flyte-core:
...
+  ## @param tfySignedUrlServer.image to set the image
+  image:
+    ## @param tfySignedUrlServer.image.repository to set the image repository
+    repository: "tfy.jfrog.io/tfy-private-images/tfySignedUrlServer"
+    ## @param tfySignedUrlServer.image.tag to set the image tag
+    tag: "v0.0.1"
+  ## @param tfySignedUrlServer.replicaCount to set the replica count
+  replicaCount: 2
```
Comment (akashg3627) [inline charts/tfy-workflow-propeller/values.yaml:109]: why do we need 2 replicas?

Comment (nikp1172) [reply charts/tfy-workflow-propeller/values.yaml:109]: this needs to be highly available

**Review by akashg3627 — APPROVED** [review]

**Pull request merged**